### PR TITLE
Adds "confirm not sent" functions

### PR DESCRIPTION
**Pull request by sjforman**
This PR is based on a branch that @jeffkole started. I: 

 * resolved merge conflicts with @andrewjtimmons' previous work
 * adjusted the `bundles` view to include `not-sent` voters
 * changed some functions in `voterService` to exclude `not-sent` voters
 * changed the logic for which buttons are enabled / disabled on the User admin page
 
This also includes some formatting changes made by running `npm run format`.

One issue I'm aware of: you can get into an odd state, where, if you have some voters marked "prepped," but not "sent," and you mark them all "not sent" (on the admin view) they will still appear on the dashboard as "prepped," because there isn't yet any other place to put them. 

You'll still be able to *mark* them "sent," but then when you reload the page they'll move back to where they were, because this is an invalid action (they can't be both "sent" and "not sent.") 

We should probably add a "not sent" column at some point, but short of that, this problem should be resolved by the next part of the project — the "Summarizing past results" part of #293, after which we'll no longer show this dashboard view for elections in the past anyway.

Happy-path test: adopt some voters, mark a few of them "prepped," and a few "sent." Go to the admin screen, click on the new button for "mark all not resolved voters `not sent.` Make sure the counts update you you expect.

Comments / feedback / suggestions / etc. welcome. Thanks!

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -433,23 +471,20 @@ router.route('/recaptcha').post(checkJwt, function(req, res) {
...
-      .catch(err => {
-        console.error(err);
-        res
-          .status(500)
-          .send(String(err))
-          .end();
-      });
-  });
-
+router.route('/user/current').get(checkJwt, function(req, res) {
```
Comment (andrewjtimmons) [inline server/server.js]: This just looks like formatting but can someone verify that to make sure im not missing a change on this?

**Comment by andrewjtimmons** [comment]
I wonder for ```One issue I'm aware of: you can get into an odd state, where, if you have some voters marked "prepped," but not "sent," and you mark them all "not sent" (on the admin view) they will still appear on the dashboard as "prepped," because there isn't yet any other place to put them.```

We could update the query for voters to be prepped to excluded anyone confirmed not sent?

However, given that future work will take care of that I dont think that needs to block this pr.

Tests and functionality look good! Nice work Jeff and Scott!

**Comment by andrewjtimmons** [comment]
+1

**Comment by sjforman** [comment]
> I wonder for `One issue I'm aware of: you can get into an odd state, where, if you have some voters marked "prepped," but not "sent," and you mark them all "not sent" (on the admin view) they will still appear on the dashboard as "prepped," because there isn't yet any other place to put them.` We could update the query for voters to be prepped to excluded anyone confirmed not sent?

Ah — good call! I've updated the display to exclude the ones marked "not sent."

**Comment by sjforman** [comment]
(I'm back from NY and getting back into this). Thanks for reviewing @andrewjtimmons! Merging.

**Comment by andrewjtimmons** [comment]
Great job!

**Pull request merged**